src/boot/lib/mlang.ml

```diff
@@ -304,7 +304,7 @@ let rec desugar_tm nss env =
   | TmSeq(fi, tms) -> TmSeq(fi, Mseq.map (desugar_tm nss env) tms)
   | TmRecord(fi, r) -> TmRecord(fi, Record.map (desugar_tm nss env) r)
   | TmRecordUpdate(fi, a, lab, b) -> TmRecordUpdate(fi, desugar_tm nss env a, lab, desugar_tm nss env b)
-  | TmUtest(fi, a, b, body) -> TmUtest(fi, desugar_tm nss env a, desugar_tm nss env b, desugar_tm nss env body)
+  | TmUtest(fi, a, b, using, body) -> TmUtest(fi, desugar_tm nss env a, desugar_tm nss env b, using, desugar_tm nss env body)
```

Comment:
I think this should be something like `Option.map (desugar_tm nss env) using` instead of just `using`, in case there's a `use` inside that part of the expression.

Reply:
Ooh nice catch, I thought I had fixed all of those.